protos/feast/serving/ServingService.proto

```diff
@@ -173,6 +177,9 @@ message Job {
     // Output only. The data format for all the files.
     // For CSV format, the files contain both feature values and a column header.
     DataFormat data_format = 6;
+    // Output only. The statistics computed over
+    // the retrieved dataset. Only available for BigQuery stores.
+    tensorflow.metadata.v0.DatasetFeatureStatisticsList dataset_feature_statistics_list = 7;
```

Comment:
After this PR, maybe after 0.6, we should definitely encapsulate these final fields (file_uris, data_format, and stats) in a dataset message.